packages/notebook-extension/schema/tracker.json

```diff
@@ -364,6 +364,24 @@
       "description": "Should timing data be recorded in cell metadata",
       "type": "boolean",
       "default": false
+    },
+    "cellNumberToRenderDirectly": {
+      "title": "Number of cells to render directly",
+      "description": "Number of cells to render directly",
+      "type": "number",
+      "default": 7
+    },
+    "renderCellOnIdle": {
+      "title": "Render cell on browser idle time",
+      "description": "Render cell on browser idle time",
+      "type": "boolean",
+      "default": true
+    },
+    "nonObservedBottomMargin": {
```

Comment:
This is more for power user that would have an understanding of how the virtual notebook works and want to optimize the rendering. e.g. you could set 1000px and the observed area would be 1000px down the bottom off the displayed notebook. All cells being added in that area would be observed and rendered. This is really a technical parameter. I am going to add more description for this params so that they will be readable in the user setting page.